libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -522,9 +522,9 @@ object MapboxRouteLineUtils {
                 val isInAClosure = closureRanges.any { it.contains(index) }
                 val congestionValue: String = when {
                     isInAClosure -> RouteConstants.CLOSURE_CONGESTION_VALUE
-                    trafficCongestion?.isEmpty() ?: true -> RouteConstants.UNKNOWN_CONGESTION_VALUE
-                    index > trafficCongestion?.size ?: 0 -> RouteConstants.UNKNOWN_CONGESTION_VALUE
-                    else -> trafficCongestion?.get(index) ?: RouteConstants.UNKNOWN_CONGESTION_VALUE
+                    trafficCongestion.isNullOrEmpty() -> RouteConstants.UNKNOWN_CONGESTION_VALUE
+                    index >= trafficCongestion.size -> RouteConstants.UNKNOWN_CONGESTION_VALUE
```

Comment:
could we cover it by tests?

Reply:
@cafesilencio do you know why in some situations the congestion list can be shorter than the distances list when both are available for a specific route leg? Is there some filtering that we do, or is this the way data comes in from the Directions API?